app/assets/javascripts/templates/visualizations/controls/histogram-tools.hbs

```diff
@@ -8,5 +8,6 @@
   <div>
     <label for="bin-size">Manual:</label>
     <input id="bin-size" class="form-control" value="{{binSize}}">
+    <button id="set-bin-size" class="btn btn-default">Set Bin Size</button>
```

Comment:
Multiple things with this line ->
1. in order to be compatible with Jim's work, it needs a log id. See the save hbs for examples.
2. the id should be 'set-bin-size-btn', as that is the precedent I've set in other tools.
3. Why is there no code associated with this button? Doesn't it's click need to be adjusted to the code that deals with hitting enter on input?